src/main/java/duke/ui/Message.java

```diff
@@ -0,0 +1,104 @@
+package duke.ui;
+
+import duke.exceptions.*;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+public class Message {
+    private static final String INDENT = "\t";
+    private static final String NEWLINE = System.lineSeparator();
+    private static final String LOGO =
+            " ____        _        " + NEWLINE
+                    + "|  _ \\ _   _| | _____ " + NEWLINE
+                    + "| | | | | | | |/ / _ \\" + NEWLINE
+                    + "| |_| | |_| |   <  __/" + NEWLINE
+                    + "|____/ \\__,_|_|\\_\\___|" + NEWLINE;
+    public static final String LINE = INDENT
+            + "__________________________________________________"
+            + "______________" + NEWLINE;
+    public static final String START_MSG =
+            "Hello from" + NEWLINE + LOGO + LINE + INDENT + "Hello! I'm Duke" + NEWLINE + INDENT
+                    + "What can I do for you?" + NEWLINE + LINE;
+    public static final String BYE_MSG =
+            LINE + INDENT + " Bye. Hope to see you again soon!" + NEWLINE + LINE;
+
+    public static String getErrorMsg(DukeException e) {
```

Comment:
I think JavaDocs comment is missing for this class. as well as some other class too